tests/test_component_as_view.py

```diff
@@ -56,9 +56,9 @@ def render_template_view(request):
         client = CustomClient(urlpatterns=[path("test_template/", render_template_view)])
         response = client.get("/test_template/")
         self.assertEqual(response.status_code, 200)
-        self.assertIn(
-            b'<input type="text" name="variable" value="TEMPLATE">',
-            response.content,
+        self.assertInHTML(
+            '<input type="text" name="variable" value="TEMPLATE">',
+            response.content.decode(),
```

Comment:
I change the assertion from `self.assertIn()` to `self.assertInHTML()` because if I remember correctly `assertInHTML` allows the HTML attributes to be in any order, and pre-processing the HTML with BS4 sometimes changed the order of the attributes.

And `assertInHTML()` accepts only strings, thus the `.decode()`.